typings/index.d.ts

```diff
@@ -2089,7 +2089,14 @@ declare module 'discord.js' {
     readonly createdTimestamp: number;
     readonly url: string;
     delete(reason?: string): Promise<void>;
+    deleteMessage(message: MessageResolvable): Promise<void>;
     edit(options: WebhookEditData): Promise<Webhook>;
+    editMessage(
+      message: MessageResolvable,
+      content: APIMessageContentResolvable,
+      options?: WebhookEditMessageOptions,
+    ): Promise<Message | object>;
+    editMessage(message: MessageResolvable, options: WebhookEditMessageOptions): Promise<Message | object>;
```

Comment:
why they are typed `Promise<Message | object>`, why `object` ?

Reply:
Did you read the description of this PR?

Reply:
I am really sorry.